internal/migrations/20241122160001_update_new_column_control_plane_count_value_for_existing_multinode_cluster_records_test.go

```diff
@@ -0,0 +1,136 @@
+package migrations
+
+import (
+	gormigrate "github.com/go-gormigrate/gormigrate/v2"
+	"github.com/go-openapi/swag"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"gorm.io/gorm"
+)
+
+var _ = Describe("updateNewColumnControlPlaneCountValueForExistingMultiNodeClusterRecords", func() {
```

Comment:
can we also test rollback function

Reply:
Do you have an idea how to make the migration fail ?